src/vue/Alert.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="uk-alert"
+    :data-uk-alert="options"
+    :class="{
+      'uk-alert-success': style === 'success',
+      'uk-alert-warning': style === 'warning',
+      'uk-alert-danger': style === 'danger',
+      'uk-alert-large': size === 'large'
+    }">
+    <a href="" class="uk-close" :class="trigger | formatClass" v-if="close"></a>
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+import UI from 'uikit'
+
+export default {
+  props: {
+    style: {
+      type: String,
+      default: ''
+    },
+    size: {
+      type: String,
+      default: ''
+    },
+    close: {
+      type: Boolean,
+      default: false
+    },
+    fade: {
+      type: Boolean,
+      default: true
+    },
+    duration: {
+      type: Number,
+      default: 200
+    },
+    trigger: {
+      type: String,
+      default: '.uk-alert-close'
+    }
+  },
+  computed: {
+    options: function () {
+      let options = {}
+      if (!this.close) return false
+      if (this.fade !== true) options.fade = this.fade
+      if (this.duration !== 200) options.duration = this.duration
+      if (this.trigger !== '.uk-alert-close') options.trigger = this.trigger
+
+      return !UI.$.isEmptyObject(options) ? JSON.stringify(options) : true
```

Comment:
Avoid using jQuery, instead check what has Vue.util or if required we can expand with our own utils.